include/common/s1_common_types.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019-present, Open Networking Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
```

Comment:
This needs fix as soon as code gets into main branch.

Reply:
Fixed.